helm/wireguard/values.yaml

```diff
@@ -66,6 +66,10 @@ service:
   port: 51820
   # -- Node port, only valid with service type: NodePort
   nodePort: 31820
+  # -- External Traffic Policy for the service
+  externalTrafficPolicy: Cluster
```

Comment:
Can you set this to nil so by default it's not included in the service definition except when a value is provided to preserve the existing behavior for people on upgrade unless they opt in to set a value.